Avoid running crosscompiled openmp test and just provide proper flags.

--- a/setupext.py
+++ b/setupext.py
@@ -48,6 +48,9 @@ def stdchannel_redirected(stdchannel, dest_filename):
 
 
 def check_for_openmp():
+    # Specifically for conda, where we know where we build, but there are
+    # cases when we cross-compile and we cannot run build checks...
+    return ["-fopenmp"], ["-fopenmp"]
     """Returns OpenMP compiler and linker flags if local setup supports
     OpenMP or [], [] otherwise
 
